File: 1952-ThreeDivisors/1952-ThreeDivisors.js
// Last updated: 3/30/2026, 2:12:08 PM
1/**
2 * @param {number} n
3 * @return {boolean}
4 */
5var isThree = function(n) {
6    let count = 0;
7    for(let i=1;i<=n;i++){
8        if(n%i==0){
9            count++;
10        }
11    }
12
13    if(count==3){
14        return true;
15    }
16    return false;
17};

File: 1952-ThreeDivisors/1952-ThreeDivisors.cpp
// Last updated: 5/10/2026, 8:59:28 AM
1class Solution {
2public:
3    bool isThree(int n) {
4        int count = 0;
5
6        for(int i=1;i<=n;i++){
7            if(n%i==0){
8                count++;
9            }
10        }
11
12        if(count==3){
13            return true;
14        }
15        return false;
16    }
17};